cypress/integration/home_page_spec.js

```diff
@@ -68,10 +92,13 @@ describe('Home page user flow', () => {
           .should('have.value',  '6')
           .get('button')
           .click()
-          .get('.pokemon-card')
+          .get('.pokemon-details-page')
           .contains('charizard')
           .get('img[src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/6.png"]')
           .should('be.visible')
     });
 
```

Comment:
Nice job testing for a second pokemon! Maybe down the line we can make a command for this too?